README.md

```diff
@@ -53,6 +53,7 @@ export EXCHANGE_ROOTPW=myrootpw
 ```
 export EXCHANGE_KEY_PW=<pass-phrase>
 make gen-key
+Note : pass-phrase can be any alphanumeric string
```

Comment:
Are you sure special characters aren't allowed?

Reply:
I need to confirm this once.